database/migrations/versions/65daf2256cbe_initial_migration.py

```diff
@@ -0,0 +1,73 @@
+"""empty message
+
+Revision ID: 65daf2256cbe
+Create Date: 2018-09-28 01:54:19.514091
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '65daf2256cbe'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'attestation',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column(
+            'method',
+            sa.Enum('PHONE', 'EMAIL', 'AIRBNB', 'FACEBOOK', 'TWITTER', name='attestationtypes'),
+            nullable=True
+        ),
+        sa.Column('eth_address', sa.String(), nullable=True),
+        sa.Column('value', sa.String(), nullable=True),
+        sa.Column('signature', sa.String(), nullable=True),
+        sa.Column('created_at', sa.DateTime(), nullable=True),
+        sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table(
+        'eth_notification_endpoint',
```

Comment:
Note: this table is for mobile notifications and not being used at the moment so would be ok to delete it. But that's fine if you prefer keeping it around in this PR.

Reply:
I thought about cleaning this up, but then I thought we might use it. Now that I think about it more it seems pretty unlikely we'll use `origin-bridge` for this functionality and it will be handled elsewhere. Would you agree? If so I'll remove the table.

Reply:
Yes. I chatted with @crazybuster - the plan is to move all the mobile back-end to the Apollo GraphQL server. So totally ok if you want to delete the eth_notification_endpoint table on bridge.